manuscript.md

```diff
@@ -2,234 +2,111 @@
+
+## What is currently limiting our understanding of species interaction networks?
+
+
+### Data
+At the moment, our understanding of the structure of ecological networks is limited by the availability of data. Although we have seen a growth in species occurrences data, this growth is much slower for ecological interactions because species interactions are challenging to sample comprehensively [@Bennett2019PotPit; @Jordano2016SamNet]  and sampling methodology matters [@deAguiar2019RevBia].
+In turn, the difficulty of sampling interactions can lead to biases in our understanding of network structure [@deAguiar2019RevBia]. This knowledge gap has motivated a variety of approaches to deal with interactions in ecological research based on assumptions that do not always hold, such as the correlation between co-occurrence and meaningful interaction strength, when it is known that co-occurrence is the only prerequisite for an interaction to occur [@Blanchet2020CooNot].
+Spatial biases in data coverage are
+prevalent at the global scale (with South America, Africa and Asia being underrepresented) and different interaction types show biases towards different biomes (or environmental conditions) [@Poisot2020EnvBia]. These 'spatial gaps' serve as a limitation to our ability to confidently make predictions when accounting for real-world environmental conditions, especially in environments for which there are no analogous data.
+
+### Methods
+This scarcity of data limits the range of computational tools than can be used by network ecologists. Most deep learning methods, for instance, are very data expensive.
+The paucity of data is compounded by a collection of biases that can be found in existing datasets. Species interaction datasets are typically dominated by food webs, pollination, and host-parasite networks [@Ings2009EcoNet; @Poisot2020EnvBia]. This could prove to be a limiting factor when trying to understand or predict networks of *under represented* interaction types or trying to integrate networks of different types [@Fontaine2011EcoEvo], especially given the structural variation of ecological networks [@Michalska-Smith2019TelEco]. This stresses the need for an integrated, flexible, and data-efficient set of computational tools which will allow us to predict ecological networks accurately from existing and imperfect datasets.  
+
+### Scale
```

Comment:
When reading this I thought it might be a good idea to include time here as well e.g. ontogeny and seasonal variation?? This would tie in nicely with the figure